docs/docker-loaders.md

```diff
@@ -0,0 +1,75 @@
+# Moving to docker based loaders
+
+### Current situation
+
+Currently, have fixed images for AWS, installing from repo/list in GCE/Azure
+Into those images we install enterprise cassandra-stress, so it would cover
+all the functionality needed
+
+Rest of the stress/load tools are installed during setup of the loader nodes (scylla-bench, cassandra-harry, gemini),
+which take time to install/compile
+
+ycsb, ndbench, nosqlbench, cdcreader and kcl are using `DockerBasedStressThread`, and served as a POC for the idea
+of using docker, working for quite a long time without any specific to docker issues.
+
+### Why
+
+We want to align all the loaders to be docker based, for the following reasons:
+1) speed up the startup of setup of loaders
+2) be able to run multiple versions of loaders at the same run
+3) better fit k8s operation for testing scylla-operator, and have pod (or multiple of them) per stress tool as needed
+4) stop manually maintaining images across multiple regions and cloud vendors
+
+### Images
+
+For most of the tools we are going to create our docker images, under `scylladb/hydra-loaders` and with
+instructions on building each one in `/docker`, we should version them by the tool version if applicable, or by date.
+the only tool we are not going to build ourselves is cassandra-stress, which would be comping from the official docker images
+of scylladb.
+
+### Finding correct cassandra-stress image
+
+Since we are going to be using scylla docker images for loaders, we need a way to find the correct docker image, based on a
+given version or sha of scylla (both for OSS and Enterprise versions), if we'll find that match base on sha would be problematic
+we'll fall back to last available release (smaller than the currently under test version, ex. when testing 5.2-dev, we'll take 5.1.x version)
```

Comment:
In any case, we will of course need a way to control that (e.g when finding issue in new release and want to isolate the issue and make sure it's not due to a new c-s version).